Game.java

```diff
@@ -0,0 +1,21 @@
+package UnitOmega13;
+
+public class Game {
+
+	Boolean gameBool1, gameBool2, gameBool3, gameBool4;
+
+	public boolean gameStart(String string1, String string2, String string3, String string4){
```

Comment:
Is it the first task in the second homework? 
If yes, change the realization
use bit operators